schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -469,6 +469,9 @@
                 },
                 "source": {
                     "$ref": "#/definitions/source"
+                },
+                "tags": {
```

Comment:
We need one inside adpContainer, minus unsupported-when-assigned tag. Perhaps to keep it simple, should we revisit the decision to not allow unsupported-when-assigned for ADPs?